protocol/triple/triple-tool/gen/generator/tripleTpl_test.go

```diff
@@ -452,22 +465,22 @@ type GreetServiceHandler interface {
 	
 }
 
-func ProvideGreetServiceHandler(pro *provider.Provider, hdlr GreetServiceHandler) error {
-	return pro.Provide(hdlr, &GreetService_ServiceInfo)
+func RegisterGreetServiceHandler(srv *server.Server, hdlr GreetServiceHandler, opts ...server.ServiceOption) error {
+	return srv.Register(hdlr, &GreetService_ServiceInfo, opts...)
 }
 
 type GreetService_GreetServer interface {
-	Send(*proto.GreetServerStreamResponse) error
+	Send(*proto.GreetResponse) error
```

Comment:
The problem with **GreetService_GreetServer** is the same as **GreetService_GreetClient**.